internals/overlord/servstate/handlers.go

```diff
@@ -346,9 +346,16 @@ func (s *serviceData) startInternal() error {
 	s.cmd.Dir = s.config.WorkingDir
 
 	// Start as another user if specified in plan.
-	uid, gid, err := osutil.NormalizeUidGid(s.config.UserID, s.config.GroupID, s.config.User, s.config.Group)
-	if err != nil {
-		return err
+	w := s.manager.workloads[s.config.Workload]
```

Comment:
Should we deal the a failure lookup here, even though in theory validation already happened ?